query-engine/prisma/src/main.rs

```diff
@@ -78,11 +72,35 @@ pub enum CliOpt {
     /// Output the DMMF from the loaded data model.
     Dmmf,
     /// Get the configuration from the given data model.
-    GetConfig(GetConfigInput),
+    GetConfig,
     /// Executes one request and then terminates.
     ExecuteRequest(ExecuteRequestInput),
 }
 
+pub fn parse_base64_string(s: &str) -> PrismaResult<String> {
+    match base64::decode(s) {
+        Ok(bytes) => String::from_utf8(bytes).map_err(|e| {
+            trace!("Error decoding {} from Base64 (invalid UTF-8): {:?}", s, e);
+
+            PrismaError::ConfigurationError("Invalid Base64".into())
+        }),
+        Err(e) => {
+            trace!("Decoding Base64 failed (might not be encoded): {:?}", e);
+            Ok(String::from(s))
+        }
+    }
+}
+
+pub fn load_datamodel_file(path: &OsStr) -> String {
+    let mut f = File::open(path).expect("Could not open datamodel file");
```

Comment:
I'm thinking, it would be nice to log what path we're trying to read the schema from, that way it's easier to debug if we have a bad env var set, for example.

Reply:
Yeah, except `OsStr` doesn't implement `Display'... Maybe with debug?

Reply:
mmh

Maybe `String::from_utf8_lossy`?

Reply:
or debug